README.rst

```diff
@@ -1,33 +1,17 @@
+.. warning::
```

Comment:
The warning doesn't seem to render correctly in Github: https://github.com/boto/boto/blob/087fe9eeb748deed2bb79123f76ee7eca01cd212/README.rst

Do we have any other layout that may make things more apparent?

Reply:
Added as a header.